templates/admin.html

```diff
@@ -356,6 +47,7 @@ <h3>Email Everyone</h3>
     <script src="/static/editor.js"></script>
     <script src="/static/marked.js"></script>
     <script type="text/javascript">
+    {# TODO jQuery seems to have a different name? #}
```

Comment:
Is this something we should worry about? :raised_eyebrow:

Reply:
That comment isn't right I think. I wrote it because the dropdowns weren't working and it was throwing an JS error in the console that it didn't before my changes. Plus my IDE complained that it couldn't find `jQuery`.

Anyways, the dropdowns are working again (thanks to @RockyTV pointing out that it wasn't a JS problem), and I think that JS script is working too, even though it still throws the error.

Going to remove the comment.